course/flow.py

```diff
@@ -705,7 +705,7 @@ def __init__(
 
     # Rounding to larger than 100% will break the percent bars on the
     # flow results page.
```

Comment:
Did you take any measures to prevent the percent bars being fed numbers larger than 100 (e.g. due to rounding)?